netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -390,6 +390,48 @@ _ebpf_sock_addr_set_redirect_context(_In_ const bpf_sock_addr_t* ctx, _In_ void*
     return return_value;
 }
 
+_IRQL_requires_max_(DISPATCH_LEVEL)
+static uint64_t
+_ebpf_sock_addr_get_current_process_start_key(
```

Comment:
same as above: we dont need to implement these here unless these implementations are different than what is present in ebpfcore.